pkg/machine/e2e/init_windows_test.go

```diff
@@ -22,7 +26,23 @@ var _ = Describe("podman machine init - windows only", func() {
 		testDir, mb = setup()
 	})
 	AfterEach(func() {
-		teardown(originalHomeDir, testDir, mb)
+		teardownCb(originalHomeDir, testDir, mb, func() {
+			wslTask := func(command string, dist string) {
+				GinkgoWriter.Println("Cleaning up using " + command + " with " + dist)
+				cmd := exec.Command(wutil.FindWSL(), command, dist)
+				session, err := Start(cmd, GinkgoWriter, GinkgoWriter)
+				if err != nil {
+					Fail(fmt.Sprintf("Unable to start session: %q", err))
+					return
+				}
+				session.Wait(defaultTimeout)
```

Comment:
you are never checking the exit code of the command, that may be fine for cleanup but it also means that it won't catch things such as invalid args.

Reply:
yeah this code was a bit sloppy that i was throwing in to check for a CI pass, will get something more reasonable in